src/config/voice.h

```diff
@@ -1 +1 @@
-const char* CONFIG_VoiceParse();
+const char* CONFIG_VoiceParse(unsigned id);
```

Comment:
Add "#pragma once"

Reply:
added include guards, to be consistent with the rest of code